xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/VideoCodec.h

```diff
@@ -211,24 +211,24 @@ namespace kodi
         m_instanceData->toAddon.reset = ADDON_Reset;
       }
 
-      inline static bool ADDON_Open(const AddonInstance_VideoCodec* instance, VIDEOCODEC_INITDATA &initData)
+      inline static bool ADDON_Open(const AddonInstance_VideoCodec* instance, VIDEOCODEC_INITDATA *initData)
```

Comment:
Well, don't wont to be annoying, but changing parameters here and at other places in this commit from reference to pointer for C++ member functions is a little over the top. References are generally to prefer over pointers in C++. Only C functions are "problematic", because plain C has no references. @AlwinEsch suggested to replace refs by pointers for the C functions in addon-instance/VideoCodec.h only. The rest can stay as it was originally.